align/pnr/manipulate_hierarchy.py

```diff
@@ -161,7 +161,7 @@ def gen_abstract_verilog_d( verilog_d):
         assert 'name' not in module
         module['name'] = module['abstract_name']
         del module['abstract_name']
-        del module['concrete_name']
+        #del module['concrete_name']
```

Comment:
The same here. However, this causes error in the other place. What I need is to make hiernode info match the placement json file.